sdk/identity/identity/integration/AzureFunctions/RunTest/host.json

```diff
@@ -0,0 +1,15 @@
+{
```

Comment:
https://github.com/Azure/azure-sdk-for-js/blob/main/eng/.docsettings.yml#L1 add `sdk/identity/identity/integration` to list of ommited paths